app/templates/application.hbs

```diff
@@ -14,7 +14,7 @@
         </div>
     </div>
 </nav>
+{{pinned-notifications}}
 <div id="content">
```

Comment:
I couldn't figure out how to toggle a class on `#content` when pinned notifications is open/closed.